app/pipe.search.ts

```diff
@@ -1,11 +1,12 @@
 import { Pipe, PipeTransform } from '@angular/core';
+import { Artist } from './component.app';
 
 @Pipe({
   name: 'search'
 })
 
 export class SearchPipe implements PipeTransform {
-  transform(pipeData, pipeModifier) {
+  transform(pipeData : Artist[], pipeModifier : string) {
```

Comment:
This was ok before because you have `noImplicitAny: false` in your tsconfig. I generally prefer to turn this off - it basically means types are optional. I turned it on in the `tsconfig-ngc.json` config so we needed this to get rid of those errors.